### ISSUE TITLE
webdriverio- Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: webdriverio:v7.16.14

### ISSUE THREAD

**Issue by padmavemulapati** [dequelabs/axe-core-npm#446]
Steps to reproduce:
Installed @axe-core/webdriverio - 4.3.3-alpha.240
Installed webdriverio - 7.16.14
and 
Installed chromedriver - 98.0.0

When trying to execute test:
```
const builder = new AxeBuilder({ client });
  try {
    const results = await builder.analyze();
```
getting the error:

```
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './lib/cjs/puppeteer/common/USKeyboardLayout' is not defined by "exports" in /Users/dq_user/Documents/webdriveriocore/node_modules/puppeteer-core/package.json
    at new NodeError (node:internal/errors:371:5)
    at throwExportsNotFound (node:internal/modules/esm/resolve:440:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:692:3)
    at resolveExports (node:internal/modules/cjs/loader:482:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/dq_user/Documents/webdriveriocore/node_modules/devtools/build/commands/performActions.js:6:28) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

**Comment by padmavemulapati** [comment]
Tried with `webdriverio - 7.14.1` , with this its working fine and making sure no effect of cascading over puppeteer.
Tried @axe-core/puppeteer- 4.3.3-alpha.240` testing with `puppeteer - 13.3.0` , its working fine.

**Comment by padmavemulapati** [comment]
Verified the fix, now webdriverio updated v7.16.14, and with this `@axe-core/webdriverio-4.3.3-alpha.242` working fine
